pycfmodel/model/resources/properties/security_group_egress_prop.py

```diff
@@ -29,11 +29,11 @@ class SecurityGroupEgressProp(Property):
     ToPort: Optional[ResolvableInt] = None
 
     def ipv4_slash_zero(self) -> bool:
-        if not self.CidrIp or not isinstance(self.CidrIp, str):
+        if not self.CidrIp:
             return False
-        return self.CidrIp.endswith("/0")
+        return str(self.CidrIp).endswith("/0")
```

Comment:
What do you think about using host mask?
```suggestion
        return self.CidrIp.hostmask == IPv4Address('0.0.255.255')
```

Reply:
Would it not be '255.255.255.255' mask value?

Reply:
oops: 
![image](https://user-images.githubusercontent.com/1830262/77343904-b7f06980-6d32-11ea-9b2b-e5f522743e29.png)

Reply:
This might break some things :(

Reply:
![image](https://user-images.githubusercontent.com/1830262/77344051-e79f7180-6d32-11ea-9028-72e3c11528dd.png)

Reply:
Added custom pre validator to use `strict=False` :)